dist/isotip.js

```diff
@@ -161,28 +153,30 @@ module.exports = {
         }
 
         var moTrigger = evt.target || evt.srcElement
+        var moTarget = self.selfOrAncestorByClass(moTrigger, 'tooltip-hover')
+        var relatedTarget = (moEvt.relatedTarget || moEvt.toElement)
 
-        if (self.hasClass(moTrigger)) {
+        // If the tooltip shouldn't autoclose, bail
+        if (self.currentTooltip && self.currentTooltip.getAttribute('data-autoclose') === 'false') {
           return
         }
 
-        // If the tooltip shouldn't autoclose, bail
-        if (self.currentTooltip && self.currentTooltip.getAttribute('data-autoclose') === 'false') {
+        if (relatedTarget && (moTarget === relatedTarget || self.hasParent(relatedTarget, moTarget))) {
```

Comment:
Looking at `relatedTarget`, that element will be the actual tooltip (in FF 48), so you'd want this check to be `relatedTarget === self.currentTooltip`